IdenfyKycPlugin/Classes/IdenfyKycProviderPlugin.swift

```diff
@@ -59,22 +59,14 @@ public class IdenfyKycProviderPlugin: NSObject, SNCKycProviderPlugin {
     
     public var platform: String = "idenfy"
     
+    private var idenfyController: IdenfyController?
```

Comment:
will they reside in memory after discarding?

Reply:
Since we instantiate the plugin once, then I guess they actually will remain in memory... Will add a fix to remedy that.

Reply:
done